diff --git a/vis.h b/vis.h
index b5a27fe..dbf254e 100644
--- a/vis.h
+++ b/vis.h
@@ -722,6 +722,8 @@ bool vis_jumplist_next(Vis*);
 /** Register specifiers. */
 enum VisRegister {
 	VIS_REG_DEFAULT,    /* used when no other register is specified */
+	VIS_REG_CLIPBOARD = VIS_REG_DEFAULT,  /* system clipboard register */
+	VIS_REG_PRIMARY = VIS_REG_DEFAULT,    /* system primary clipboard register */
 	VIS_REG_ZERO,       /* yank register */
 	VIS_REG_AMPERSAND,  /* last regex match */
 	VIS_REG_1,	    /* 1-9 last sub-expression matches */
@@ -734,8 +736,6 @@ enum VisRegister {
 	VIS_REG_8,
 	VIS_REG_9,
 	VIS_REG_BLACKHOLE,  /* /dev/null register */
-	VIS_REG_CLIPBOARD,  /* system clipboard register */
-	VIS_REG_PRIMARY,    /* system primary clipboard register */
 	VIS_REG_DOT,        /* last inserted text, copy of VIS_MACRO_OPERATOR */
 	VIS_REG_SEARCH,     /* last used search pattern "/ */
 	VIS_REG_COMMAND,    /* last used :-command ": */
